spec/models/dish_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Dish, type: :model do
+  describe 'DB table Dish' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :name }
+    it { is_expected.to have_db_column :description }
+    it { is_expected.to have_db_column :price }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :name }
+    it { is_expected.to validate_presence_of :price }
```

Comment:
I assume that we don't validate that a dish actually belongs to a menu. That surprises me a bit. Is there a use case where there would be a need for a dish that is NOT associated with a menu?